plugins/services/src/js/components/modals/ServiceResumeModal.js

```diff
@@ -100,7 +101,9 @@ class ServiceResumeModal extends React.PureComponent {
     }
 
     return (
-      <h4 className="text-align-center text-danger flush-top">{errorMsg}</h4>
+      <Trans render="h4" className="text-align-center text-danger flush-top">
```

Comment:
This won't work. If you want to translate `"Must be bigger than or equal to 0"` but then fallback to the error message if it doesn't `== " error.min"` then you'll need to mark and translated `Must be bigger than or equal to 0` in `getActionModalReadableError` and not use `Trans` here.